README.md

```diff
@@ -234,4 +236,31 @@ Also cite the RIR and sound event databases that SpatialScaper uses.
   journal={arXiv preprint arXiv:1612.01840},
   year={2016}
 }
+
+@article{gotz2021dataset,
+  title={A dataset of higher-order Ambisonic room impulse responses and 3D models measured in a room with varying furniture},
+  author={G{\"o}tz, Georg and Schlecht, Sebastian J and Pulkki, Ville},
+  journal={2021 Immersive and 3D Audio: from Architecture to Automotive (I3DA)},
+  pages={1--8},
+  year={2021},
+  publisher={IEEE}
+}
+
+@article{schneiderwind2019data,
+  title={Data set: Eigenmike-DRIRs, KEMAR 45BA-BRIRs, RIRs and 360◦ pictures captured at five positions of a small conference room},
+  author={Schneiderwind, Christian and Neidhardt, Annika and Klein, F and Fichna, S},
+  journal={45th Annual Conference on Acoustics (DAGA), Rostock, Germany},
+  year={2019}
+}
+
+@article{chesworth2024room,
+  title={Room Impulse Response Dataset of a Recording Studio with Variable Wall Paneling Measured Using a 32-Channel Spherical Microphone Array and a B-Format Microphone Array},
+  author={Chesworth, Grace and Bastine, Amy and Abhayapala, Thushara},
+  journal={Applied Sciences},
+  volume={14},
+  number={5},
+  pages={2095},
+  year={2024},
+  publisher={Multidisciplinary Digital Publishing Institute}
+}
```

Comment:
@karinnacvijanovich please add the three rooms and their associated details to the table in lines 77-89